generate-manifest

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import os
+import shutil
+import subprocess
+import sys
+import urllib.parse
+
+BASE_PATH = os.path.realpath(os.path.dirname(__file__))
+MANIFEST_TEMPLATE_PATH = '{}/org.learningequality.Kolibri.Content.json.in'.format(BASE_PATH)
+KOLIBRI_HOME_PATH = '{}/kolibri_home'.format(BASE_PATH)
+CONTENT_JSON_PATH = '{}/content/content.json'.format(KOLIBRI_HOME_PATH)
+_KOLIBRI_FLATPAK_REF = None
```

Comment:
What's the purpose of this variable which is never assigned to?

Reply:
It's used as cache in `get_kolibri_flatpak_ref`

Reply:
No it's not, that function doesn't assign to it.

Reply:
Now yeah, updated.